pom.xml

```diff
@@ -645,7 +645,7 @@
                             <includeModule>profiles</includeModule>
                             <includeModule>protocols</includeModule>
                             <includeModule>regions</includeModule>
-                            <includeModule>sdk-core</includeModule>
+                            <!-- <includeModule>sdk-core</includeModule> -->
```

Comment:
This doesn't look right.

Reply:
It's needed since `retryStrategy(RetryMode)` and `retryStrategy(Consumer<RetryStrategy.Consumer>)` are not default methods but instead abstract. I'm like 98.5% confident that this is backwards compatible so I feel like it's a false positive but still. The plan is to remove it after we release this change.

Without this the build fails with:

```
[ERROR] Failed to execute goal com.github.siom79.japicmp:japicmp-maven-plugin:0.15.6:cmp (default) on project sdk-core: There is at least one incompatibility: software.amazon.awssdk.core.client.config.ClientOverrideConfiguration$Builder.retryStrategy(software.amazon.awssdk.core.retry.RetryMode):METHOD_NOW_ABSTRACT,software.amazon.awssdk.core.client.config.ClientOverrideConfiguration$Builder.retryStrategy(java.util.function.Consumer):METHOD_NOW_ABSTRACT -> [Help 1]
```